src/main/resources/GoogleApps/Translations.de.xml

```diff
@@ -66,8 +66,8 @@ GoogleApps.GoogleAppsConfigClass_scope=Autorisierungen
 GoogleApps.GoogleAppsConfigClass_scope.hint=Welche Information sollte dieses Wiki Google abfragen?
 GoogleApps.GoogleAppsConfigClass_scope_identity=Vor- und Nachname.
 GoogleApps.GoogleAppsConfigClass_scope_email=Emailadresse.
-GoogleApps.GoogleAppsConfigClass_scope_gdoc=Google Docs lesen und speichern.
-GoogleApps.GoogleAppsConfigClass_scope_readAvatar=Benutzerprofilbild
```

Comment:
It's strongly recommended to not delete translation keys. They should be moved at the bottom of the page, in a `deprecated` area. https://dev.xwiki.org/xwiki/bin/view/Community/DevelopmentPractices#HTranslationPropertyDeprecations.

Reply:
fixed. thanks.